app/Leaf/Model/GlobalSession.php

```diff
@@ -17,7 +17,7 @@ public function getSession(string $csrf): array
     {
         $vars = array(':session' => $csrf);
         $sql = 'SELECT `csrf`, `session`, `established`
-                FROM `sessions`
+                FROM `global_sessions`

```

Comment:
I may have missed it but where is this table getting created?

Reply:
It's going to be created in the national leaf launchpad